muweb-socket/src/socket.ts

```diff
@@ -8,7 +8,7 @@ export class MuWebSocket implements MuSocket {
     private _closed:boolean = false;
 
     private _url:string;
-    private _reliableSocket:WebSocket;
+    private _reliableSocket!:WebSocket;
```

Comment:
Make this `WebSocket|null`

Reply:
do we really need two different bottom values?